internal/dinosql/parser.go

```diff
@@ -340,20 +340,34 @@ func validateQueryName(name string) error {
 		isLetter := unicode.IsLetter(c) || c == '_'
 		isDigit := unicode.IsDigit(c)
 		if i == 0 && !isLetter {
-			return fmt.Errorf("invalid query name: %q", name)
+			return fmt.Errorf("invalid query name %q", name)
 		} else if !(isLetter || isDigit) {
-			return fmt.Errorf("invalid query name: %q", name)
+			return fmt.Errorf("invalid query name %q", name)
 		}
 	}
 	return nil
 }
 
-func parseMetadata(t string) (string, string, error) {
+type CommentStyle int
+
+const (
+	MySQLCommentStyle CommentStyle = iota
+	PGCommentStyle
+)
+
+func ParseMetadata(t string, commentStyle CommentStyle) (string, string, error) {
```

Comment:
> It looks like the available methods exposed by sqlparser only support /* */ comments for extraction. 

That's fine for now. Can we add a comment to this function explaining that the MySQL parser doesn't support `--` and `#` comments?